article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -61,7 +63,9 @@ case class PageData(
     contentType: Option[String],
     commissioningDesks: Option[String],
     subMetaLinks: SubMetaLinks,
-
+    sentryHost: Option[String],
+    sentryPublicApiKey: Option[String],
+    isDev: Boolean,
```

Comment:
This flag might be useful to dotcom-rendering in some ways, however I don't think it's useful for Sentry. Given that `dotcom-rendering` in DEV mode hits the PROD frontend endpoints, this will lead to dev errors being sent to Sentry.

Reply:
Very good point, I could put something in my corresponding branch on dotcom-rendering which overrides `isDev` to `true`  when using the dev server.

Reply:
Okay, I think we should make it clear whether this flag is used to indicate whether frontend is running in dev mode, or dotcom-rendering is running in dev mode, or some conflation of the two. Can we chat face to face next week to help me better understand what it’s being used for?